data/basal.go

```diff
@@ -0,0 +1,166 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
```

Comment:
Is this necessary here?  Doesn't look like it is used.  `validator` is used in the init, but where is that pulled in from?  Perhaps it is just the package rename to `valid` that is confusing me.